services/navigation/builtin/builtin.go

```diff
@@ -231,15 +231,24 @@ func (svc *builtIn) SetMode(ctx context.Context, mode navigation.Mode, extra map
 	return nil
 }
 
-func (svc *builtIn) Location(ctx context.Context, extra map[string]interface{}) (*geo.Point, error) {
+func (svc *builtIn) Location(ctx context.Context, extra map[string]interface{}) (*geo.Point, float64, error) {
```

Comment:
This will also be useful when we inevitable also return Altitude.

For this specific PR I'm ok with either. Returning 3 things (really two things plus maybe-error) is on the OK side of borderline for me, while returning 4 things is wholly unacceptable.